### PR TITLE
Adjusted in help messages for consistency

### PR DESCRIPTION
**Pull request by SphinxC0re**
Also adjusted the display of command help messages by adding at least one space between a command and its description

### CODE REVIEW

**Comment by SphinxC0re** [comment]
I got another suggestion: What about adding a ``` - ``` before every helpstring

**Comment by madmaxoft** [comment]
NO! The helpstring is data, the dash is formatting. Keep the two separated. The dash should be added by the help command handler. There should be no dash in the Info.lua description nor in the forum and github readmes

Julian Laubstein <notifications@github.com> wrote:

>I got another suggestion: What about adding a ``` - ``` before every
>helpstring
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/cuberite/cuberite/pull/2678#issuecomment-158817835

-- 
Sent from my Android phone with K-9 Mail. Please excuse my brevity.

**Comment by SphinxC0re** [comment]
So? The dash is just displayed by the printHelp method.

**Comment by SphinxC0re** [comment]
Oh :laughing: I didn't mention that by "adding" I meant adding the dash when displaying the actual help :laughing:

File: src/Server.cpp [hunk 1]
```diff
@@ -597,7 +597,7 @@ void cServer::PrintHelp(const AStringVector & a_Split, cCommandOutputCallback &
 	for (AStringPairs::const_iterator itr = Callback.m_Commands.begin(), end = Callback.m_Commands.end(); itr != end; ++itr)
 	{
 		const AStringPair & cmd = *itr;
-		a_Output.Out(Printf("%-*s%s\n", static_cast<int>(Callback.m_MaxLen), cmd.first.c_str(), cmd.second.c_str()));
+		a_Output.Out(Printf("%-*s%s\n", static_cast<int>(Callback.m_MaxLen), cmd.first.c_str(), (" - " + cmd.second).c_str()));
```
Comment (madmaxoft) [inline src/Server.cpp]: This is stupid, the dash should have gone in the format string instead.

**Comment by tigerw** [comment]
xoft irl:

```
 Cause you had a bad day
 You're taking one down
 You sing a sad song just to turn it around
 You say you don't know
 You tell me don't lie
 You work at a smile and you go for a ride
 You had a bad day
 The camera don't lie
 You're coming back down and you really don't mind
 You had a bad day
 You had a bad day
```

File: src/Server.cpp (hunk 1 above)
Comment (SphinxC0re) [inline src/Server.cpp]: Why exactly? This looks good and it's readable

Comment (SphinxC0re) [inline src/Server.cpp]: Okay... silly me. I don't know much about format strings.  Is there any documentation you can give me?

Comment (SphinxC0re) [inline src/Server.cpp]: Fixed in 5d5f5c9fbaa7a6d4f567a26a4b39749dee356a49

**Pull request merged**